.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,41 @@
+name: E4E Pipeline
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get E4E package version
+        id: npm-version
+        uses: nyaa8/package-version@v1.0.3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
+        run: npm ci
+        env:
+          NODE_OPTIONS: --max_old_space_size=4096
+
+      - name: Package VSIX & Upload Artifact
+        run: yarn package
```

Comment:
Do you need to install yarn first?

Reply:
Also just a note, do we need yarn audit?

Reply:
[GH Actions includes yarn by default :D](https://docs.github.com/en/codespaces/setting-up-your-codespace/configuring-codespaces-for-your-project#using-the-default-configuration)

I can throw in yarn audit, why not? :)